tests/eqlpy/eqldjango_test.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+import os
+from eqlpy.eqldjango import *
+from datetime import date
+
+
+class EqlDjangoTest(unittest.TestCase):
```

Comment:
These tests don't save anything, but they only create a JSON value for encryption, and check the values in the JSON, then convert back to Python value and check they are identical